securedrop/db.py

```diff
@@ -77,6 +77,8 @@ class Source(Base):
     flagged = Column(Boolean, default=False)
     last_updated = Column(DateTime, default=datetime.datetime.utcnow)
     star = relationship("SourceStar", uselist=False, backref="source")
+    submissions_delete = relationship("Submission", cascade="delete")
+    replies_delete = relationship("Reply", cascade="delete")
```

Comment:
How does this interact with the backrefs to `Source` that are defined on `Submission` and `Reply`? Seems like a cleaner approach might be to set up the cascade on the existing backrefs (see the [SQLAlchemy documentation on cascades](http://docs.sqlalchemy.org/en/rel_0_9/orm/cascades.html)).